src/status_im/contexts/wallet/send/from/view.cljs

```diff
@@ -23,10 +24,17 @@
 
 (defn- render-fn
   [item]
-  (let [network-details (rf/sub [:wallet/network-details])]
+  (let [network-details     (rf/sub [:wallet/network-details])
+        network-short-names (map #(network-utils/network->short-name %)
+                                 (:network-preferences-names item))
+        prefix              (when (< (count network-short-names) 3)
+                              (network-utils/short-names->network-preference-prefix network-short-names))
+        address             (str prefix (:address item))]
```

Comment:
The entire computation of `address` can be a subscription. The advantages are: simplify the view code and we cover this logic with a unit test for the sub, so overall higher quality. We might get caching from re-frame if two or more items have the same `:network-preferences-names`.

Reply:
Done